servicebroker/test/index_test.js

```diff
@@ -7,13 +7,11 @@ var path = require('path');
 var uuid = require('uuid');
 
 // This agent refers to PORT where program is runninng.
-var catalog = JSON.parse(fs.readFileSync(path.join(__dirname, '../config/catalog.json'), 'utf8'));
 var settings = JSON.parse(
   fs.readFileSync(path.join(__dirname, '../config/settings.json'), 'utf8'));
 var auth = new Buffer(settings.username + ":" + settings.password).toString('base64')
 
-// UNIT test begin
-describe("RESTful API test suite", function() {
+describe("Invalid path for RESTful API", function() {
```

Comment:
normally you only have 1 top level describe that everything nests under.
describe('routing', function() {
  before(function() {
   start server
  })
  after(function() {
  stop server
 })

 context('with an invalid path', ....)
 describe('authorization', function() ....)
})

We don't need to start and stop the server for each test since we don't change the configuration.